app/config/question-bank.js

```diff
@@ -3941,14 +3934,14 @@ const questionBank = {
             },
             {
               type: 'REGEX',
-              // no special chars, only numbers
-              regex:  /^\d*(\.\d+)?$/,
+              // no special chars, only numbers, commas and dots
+              regex:  /^[0-9,.]+$/,
               error: 'Estimated power capacity must be a number, like 10'
             },
             {
+              // number with up to 2 decimal places and commas for thousands
               type: 'REGEX',
-              // only 2 decimal places
-              regex:  /^(\d{1,2}(\.\d{1,2})?)$/,
+              regex:  /^(\d{1,3}(,\d{3})*(\.\d{1,2})?)$/,
```

Comment:
The first regex checks for the correct format (only numbers, commas and decimal points in the right way, so cant be ,,5454 accepted)
The second regex would then only need to check for 2 decimal places after the number. So it would only check for if a decimal point is there, and if there are 2 numbers or less behind it. E.g 234.567 should not be accepted but 234.56 should be
At the moment, the second regex looks like its trying to do both of these checks at once, which isn't really needed

Reply:
In the previous version, users were not allowed to include commas while entering numbers. However, the new update includes a provision that allows users to input numbers with 1-3 digits to the far left, and if desired, a comma in the middle and we use dots for decimal number.

Reply:
I just realised this cause issue when I type 2234.56 without comma.